extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -234,3 +236,71 @@
   margin-bottom: 20px !important;
   color: var(--i-amphtml-label-text-color) !important;
 }
+
+/** For amp-story-outlink-page-attachment-v2 experiment elements. */
+.i-amphtml-story-outlink-page-attachment-arrow {
+  display: block !important;
+  height: 32px !important;
+  width: 32px !important;
+  cursor: pointer !important;
+  position: relative;
+}
+
+.i-amphtml-story-outlink-page-open-attachment-bar-left {
+  background: white !important;
+  left: 6px !important;
+  animation: open-attachment-bar-left 0.3s cubic-bezier(0.4, 0.0, 0.2, 1) both !important;
+}
+
+.i-amphtml-story-outlink-page-open-attachment-bar-right {
+  background: white !important;
+  right: 6px !important;
+  animation: open-attachment-bar-right 0.3s cubic-bezier(0.4, 0.0, 0.2, 1) both !important;
+}
+
+.i-amphtml-story-outlink-page-attachment-outlink-chip {
+  display: flex !important;
+  position: relative !important;
+  padding: 10px 14px 10px 6px !important;
+  margin: 0 0 20px 0 !important;
+  height: 16px !important;
+  max-width: 243px !important;
+  border-radius: 30px !important;
+  background: white !important;
+  color: var(--cta-accent-color, black) !important;
+  place-items: center !important;
+}
+
+.i-amphtml-story-outlink-page-attachment-img {
+  height: 24px !important;
+  width: 24px !important;
+  vertical-align: middle !important;
+  background-size: contain !important;
```

Comment:
Why did you pick "contain"? I would have expected "cover"

Reply:
It was the way we styled the inline-attachment images but I looked into it and cover is a better option here. Thank you!